Example/SFFeatureFlag/FirebaseRemoteConfigService.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  FirebaseRemoteConfigService.swift
+//  SFFeatureFlag
+//
+//  Created by Maja Sapunova on 17.6.21.
+//  Copyright © 2021 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import FirebaseCore
+import FirebaseRemoteConfig
+import SFFeatureFlag
+
+class FirebaseRemoteConfigService: RemoteConfigServiceProtocol {
+   
+    //MARK: - Properties
+    private var remoteConfig: RemoteConfig?
+    
+    //MARK: - RemoteConfigServiceProtocol
+    func setup() {
+        FirebaseApp.configure()
+        
+        remoteConfig = RemoteConfig.remoteConfig()
+        
+        let settings = RemoteConfigSettings()
+        
+        //WARNING: Use the minimumFetchInterval only for debug builds
+        settings.minimumFetchInterval = 0
+        
+        remoteConfig?.configSettings = settings
+        
+        setDefaults()
+    }
+    
+    func fetchRemoteConfig(_ completionHandler: @escaping RemoteConfigCompletion) {
+        RemoteConfig.remoteConfig().fetch(completionHandler: { status, error in
```

Comment:
Not sure if ```remoteConfig``` keeps ref to the escaping closure, but maybe worth checking if we need weak ref to self here,

Reply:
Added weak self to the escaping closure